setup.sh

```diff
@@ -136,6 +136,7 @@ install_libpng() {
 install_packages() {
   npm install
   bundle
+  ./node_modules/.bin/jspm install
```

Comment:
Seeing as you've now made the bash script above, why don't you replace these three lines buy invoking that script? Will also be more DRY.